cookbooks/bcpc-hadoop/recipes/graphite_disk_wrapper.rb

```diff
@@ -1,19 +1,24 @@
-# vim: tabstop=2:shiftwidth=2:softtabstop=2 
+# vim: tabstop=2:shiftwidth=2:softtabstop=2
 # assign prepare a directory structure for graphite to install itself to
-reservation_requests = node[:bcpc][:hadoop][:disks][:reservation_requests]
+ruby_block 'graphite-directory-structure' do
+  block do
+    reservation_requests =
+      node.run_state[:bcpc_hadoop_disks][:reservation_requests]
```

Comment:
Depending on how my comments on bcpc-hadoop/attributes/disk.rb are addressed, this may need to be changed as well